rust-runtime/aws-smithy-http-server/src/plugin/mod.rs

```diff
@@ -188,3 +244,222 @@ where
         <Pl as Plugin<Ser, Op, T>>::apply(self, inner)
     }
 }
+
+/// A HTTP plugin is a plugin that acts on the HTTP request before it is deserialized, and acts on
```

Comment:
> A HTTP plugin is a plugin

I think this is misleading and why perhaps we should call it `HttpMarker` or something. This description would make sense if it was a super trait of `Plugin`.

Reply:
Why is it misleading?

Reply:
I see. Ok, will change.